ouroboros-network-framework/test/Test/Ouroboros/Network/Server2.hs

```diff
@@ -256,7 +256,7 @@ type ConnectionState_ muxMode peerAddr m a b =
                        m
 
 withInitiatorOnlyConnectionManager
```

Comment:
Could this commit be squashed with the previous one?

Reply:
Could you provide which commit, there's not enough context to find out.

Reply:
23d3c2be535340cbb472b2cc8dc2e25b8b5e0cf5 -  Address review comments